Formula/cod.rb

```diff
@@ -0,0 +1,18 @@
+class Cod < Formula
+  desc "Completion daemon for bash/fish/zsh"
+  homepage "https://github.com/dim-an/cod"
+  url "https://github.com/dim-an/cod/archive/9096c4bd0b4a34121f35c323e174a139c30aa8f6.tar.gz"
+  sha256 "dfc8de77d33d3a4e4af46eda3b61e4dbbeddf099d8f85bfc0b4c4a2d693b425c"
```

Comment:
```suggestion
  url "https://github.com/dim-an/cod/archive/v0.0.0.tar.gz"
  sha256 "6f773358f363141dcd1bbdffaaa5dea0a40ae7a84d5fc55fd453302de8493c20"
```
Let's use the tarball from a release tag.

Reply:
Right now the release tarball is pretty out of date. I have an issue open on cod to add new releases and the maintainer is working on it there.

Should we wait to merge this until he has a new release out?

Reply:
We only build software from tagged releases, so we either use the existing one or wait for a newer tag.

Reply:
Got it, let's wait for a new tag then. Fine if we keep this PR open while the issue on the cod repo is resolved?

Reply:
Fine by me.